src/devices/UFireIse/UFireIse.cs

```diff
@@ -0,0 +1,391 @@
+        /// When there are high and low calibration points, the device will automatically use them to adjust readings.To disable dual-point adjustment, call reset to remove all calibration data.
+        /// </summary>
+        /// <returns>The measured value (reading value) for calibrate the low value</returns>
+        public ElectricPotential GetCalibrateLowReading()
+        {
+            return new ElectricPotential(ReadRegister((byte)Register.ISE_CALIBRATE_READLOW_REGISTER), UnitsNet.Units.ElectricPotentialUnit.Millivolt);
+        }
+
+        /// <summary>
+        ///  Resets all the stored calibration information
+        /// </summary>
+        public void Reset()
+        {
+            WriteRegister((byte)Register.ISE_CALIBRATE_SINGLE_REGISTER, null);
+            WriteRegister((byte)Register.ISE_CALIBRATE_REFHIGH_REGISTER, null);
+            WriteRegister((byte)Register.ISE_CALIBRATE_REFLOW_REGISTER, null);
+            WriteRegister((byte)Register.ISE_CALIBRATE_READHIGH_REGISTER, null);
+            WriteRegister((byte)Register.ISE_CALIBRATE_READLOW_REGISTER, null);
+        }
+
+        /// <summary>
+        /// Sets all the values, in mV, for dual point calibration and saves them in the devices's EEPROM
+        /// </summary>
+        /// <param name="refLow">the reference low point. Range: -1024 mV to 1024 mV</param>
+        /// <param name="refHigh">the reference high point. Range: -1024 mV to 1024 mV</param>
+        /// <param name="readLow">the measured low point. Range: -1024 mV to 1024 mV</param>
+        /// <param name="readHigh">the measured high point. Range: -1024 mV to 1024 mV</param>
+        public void SetDualPointCalibration(ElectricPotential refLow, ElectricPotential refHigh, ElectricPotential readLow, ElectricPotential readHigh)
+        {
+            if (refLow.Millivolts >= -1024 && refLow.Millivolts <= 1024 &&
+                refHigh.Millivolts >= -1024 && refHigh.Millivolts <= 1024 &&
+                readLow.Millivolts >= -1024 && readLow.Millivolts <= 1024 &&
+                readHigh.Millivolts >= -1024 && readHigh.Millivolts <= 1024)
+            {
+                WriteRegister((byte)Register.ISE_CALIBRATE_REFLOW_REGISTER, Convert.ToSingle(refLow.Millivolts));
+                WriteRegister((byte)Register.ISE_CALIBRATE_REFHIGH_REGISTER, Convert.ToSingle(refHigh));
+                WriteRegister((byte)Register.ISE_CALIBRATE_READLOW_REGISTER, Convert.ToSingle(readLow));
+                WriteRegister((byte)Register.ISE_CALIBRATE_READHIGH_REGISTER, Convert.ToSingle(readHigh));
+            }
+        }
+
+        /// <summary>
+        /// Changes the default I2C address
+        /// </summary>
+        /// <param name="i2cAddress">The new I2C address</param>
+        public void SetI2CAddress(byte i2cAddress)
```

Comment:
I assume after you do that you won't be able to communicate to the device, right? If so then rename this to SetI2cAddressAndDispose (and call Dispose)